app/views/layouts/base.html.erb

```diff
@@ -45,7 +45,7 @@
   <%= javascript_include_tag_without_preload 'respond.min' %>
   <![endif]-->
 
-  <% if IdentityConfig.store.newrelic_browser_key.present? && IdentityConfig.store.newrelic_browser_app_id.present? %>
+  <% if IdentityConfig.store.newrelic_browser_app_id.present? %>
   <%= render 'shared/newrelic/browser_instrumentation' %>
```

Comment:
while we're here, can we indent the body of this if statement?
```suggestion
    <%= render 'shared/newrelic/browser_instrumentation' %>
```

Reply:
> while we're here, can we indent the body of this if statement?

Updated in 2c49f2f0d.